src/components/molecules/DotsDropdown/DotsDropdown.tsx

```diff
@@ -20,18 +20,26 @@ interface DotsDropdownProps {
     | 'bottomRight'
     | undefined;
   withPadding?: boolean;
+  isTransparent?: boolean;
+  trigger?: ('click' | 'hover' | 'contextMenu')[];
 }
 
-const DotsDropdown: React.FC<DotsDropdownProps> = ({items, placement = 'bottom', withPadding = true}) => {
+const DotsDropdown: React.FC<DotsDropdownProps> = ({
+  items,
+  placement = 'bottom',
+  withPadding = true,
+  isTransparent = true,
```

Comment:
Maybe it will be better to allow passing `style` instead? When it will not have to be transparent, it may need to be of any color

Reply:
done